examples/scott.m31

```diff
@@ -2,21 +2,21 @@
 
 (* We assume that there is a type [D] equal to its function space. *)
 constant D : Type
-constant scott : (D → D) ≡ D
-let dana = sym Type (D → D) D scott
-now hints = add_hints [dana, scott]
+constant D_reflexive : D ≡ (D → D)
+now hints = add_hints [D_reflexive, symmetry D_reflexive]
```

Comment:
Not so far. A very early version of Andromeda (that had built-in hints) always searched for a hint and its symmetric version. But it was never clear how useful that was.